package.json

```diff
@@ -106,6 +105,6 @@
     "configPath": "tests/dummy/config"
   },
   "pr-bumper": {
-    "coverage": 82.26
+    "coverage": 76.34
```

Comment:
The coverage numbers being reported were actually not accurate. I upgraded `ember-cli-code-coverage` and pinned `ember-code-snippets` to allow a more accurate number and reset the baseline here.